opm/parser/eclipse/Deck/DeckOutput.hpp

```diff
@@ -28,7 +28,8 @@ namespace Opm {
 
     class DeckOutput {
     public:
-        explicit DeckOutput(std::ostream& s);
+        DeckOutput(std::ostream& s, int precision = 10);
```

Comment:
This constructor should (probably) still be marked `explicit`.  A constructor that is callable with a single argument can be used for implicit conversion.

Reply:
OK - `explcit` taken back in.